src/App.tsx

```diff
@@ -1,29 +1,55 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
-import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
+import { getNumbers, TOTAL_ITEMS, SELECTOR_OPTIONS } from './utils';
+// import { values } from 'cypress/types/lodash';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42).map(n => `Item ${n}`);
+const items = getNumbers(1, TOTAL_ITEMS).map(n => `Item ${n}`);
+
+function useItems(itemsArr: string[], currentPage: number, perPage: number) {
+  const useItemsList: string[] = [...itemsArr];
+
+  return useItemsList.splice((currentPage - 1) * perPage, perPage);
+}
 
 export const App: React.FC = () => {
+  const [currentPage, setCurrentPage] = useState<number>(1);
```

Comment:
```suggestion
  const [currentPage, setCurrentPage] = useState(1);
```
you don't need state type if you place here default value